extensions/amp-animation/amp-animation.md

```diff
@@ -25,278 +25,92 @@ limitations under the License.
-
-For more information on `var()`, see the [`var()` and `calc()` section](#var-and-calc-expressions).
-
-### Timing properties
-
-Top-level animation and animation components may contain timing properties. These properties are defined in detail in the
-[AnimationEffectTimingProperties](https://www.w3.org/TR/web-animations/#dictdef-animationeffecttimingproperties) of the Web Animation spec. The set of properties allowed here includes:
-
-<table>
-  <tr>
-    <th class="col-twenty">Property</th>
-    <th class="col-twenty">Type</th>
-    <th class="col-twenty">Default</th>
-    <th>Description</th>
-  </tr>
-  <tr>
-    <td><code>duration</code></td>
-    <td>time</td>
-    <td>0</td>
-    <td>The animation duration. Either a numeric value in milliseconds or a CSS time value, e.g. `2s`.</td>
-  </tr>
-  <tr>
-    <td><code>delay</code></td>
-    <td>time</td>
-    <td>0</td>
-    <td>The delay before animation starts executing. Either a numeric value in milliseconds or a CSS time value, e.g. `2s`.</td>
-  </tr>
-  <tr>
-    <td><code>endDelay</code></td>
-    <td>time</td>
-    <td>0</td>
-    <td>The delay after the animation completes and before it's actually considered to be complete. Either a numeric value in milliseconds or a CSS time value, e.g. `2s`.</td>
-  </tr>
-  <tr>
-    <td><code>iterations</code></td>
-    <td>number or<br>"Infinity" or<br>"infinite"</td>
-    <td>1</td>
-    <td>The number of times the animation effect repeats.</td>
-  </tr>
-  <tr>
-    <td><code>iterationStart</code></td>
-    <td>number/CSS</td>
-    <td>0</td>
-    <td>The time offset at which the effect begins animating.</td>
-  </tr>
-  <tr>
-    <td><code>easing</code></td>
-    <td>string</td>
-    <td>"linear"</td>
-    <td>The <a href="https://www.w3.org/TR/web-animations/#timing-function">timing function</a> used to scale the time to produce easing effects.</td>
-  </tr>
-  <tr>
-    <td><code>direction</code></td>
-    <td>string</td>
-    <td>"normal" </td>
-    <td>One of "normal", "reverse", "alternate" or "alternate-reverse".</td>
-  </tr>
-  <tr>
-    <td><code>fill</code></td>
-    <td>string</td>
-    <td>"none"</td>
-    <td>One of "none", "forwards", "backwards", "both", "auto".</td>
-  </tr>
-</table>
-
-All timing properties allow either a direct numeric/string values or CSS values. For instance, "duration" can be specified as `1000` or `1s` or `1000ms`. In addition, `calc()` and `var()` and other CSS expressions are also allowed.
-
-An example of timing properties in JSON:
-
-```text
-{
-  ...
-  "duration": "1s",
-  "delay": 100,
-  "endDelay": "var(--end-delay, 10ms)",
-  "easing": "ease-in",
-  "fill": "both"
-  ...
-}
+```html
+<amp-animation layout="nodisplay">
+  <script type="application/json">
+    [
+      {
+        "selector": ".target-class",
+        "duration": 1000,
+        "keyframes": {"opacity": 1}
+      },
+      {
+        "selector": ".target-class",
```

Comment:
Let's rename selectors here to `.target1` above and `.target2` here, since the sentence above says "multiple elements".

Reply:
great suggestion, thanks